recipes/vcf-validator/build.sh

```
@@ -1,11 +1,16 @@
-mkdir -p ${PREFIX}/bin
-# Check if linux or osx
+# Install additional dependencies
```

Shellcheck

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)